backend/Dockerfile.buildapi

```diff
@@ -0,0 +1,45 @@
+# 0. Generate client code (go & json) from API protocol buffers
+FROM golang:1.13.1-stretch as generator
+ENV KFP_VERSION 1.0.0
+ENV GRPC_GATEWAY_VERSION v1.9.1
+ENV GO_SWAGGER_VERSION v0.21.0
```

Comment:
I think we were using https://github.com/kubeflow/pipelines/blob/5725bb85ed875650f209b48077db8b11aed64cd0/WORKSPACE#L126

v0.18.0.

This might be the reason of many diffs in this PR.

Reply:
I will try this, but had a lot of issues with the different versions to get it to work out. Never truly understood why but slightly change some versions to solve it.